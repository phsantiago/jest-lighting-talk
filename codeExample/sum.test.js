import sum from './sum'

describe("one plus two", () => {
  it("adds up to three", () => {
    expect(sum(1, 2)).toMatchSnapshot()
  })
})
