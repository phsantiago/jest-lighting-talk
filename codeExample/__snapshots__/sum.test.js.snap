// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`one plus two adds up to three 1`] = `3`;
